packages/framework/bundle/layerselector2/bundle.js

```diff
@@ -42,18 +45,6 @@ Oskari.clazz.define("Oskari.mapframework.bundle.layerselector2.LayerSelectorBund
         }, {
             "type": "text/javascript",
             "src": "../../../../bundles/framework/layerselector2/view/PublishedLayersTab.js"
-        }, {
-            "type": "text/javascript",
-            "src": "../../../../bundles/framework/layerselector2/request/ShowFilteredLayerListRequest.js"
-        }, {

```

Comment:
Removing requests should be documented in the changelog and the corresponding md-documents should be removed. Also the possibility of showing a filtered list programmatically is removed in this PR? This is used at least in publisher to show only publishable layers for the user and analysis to show only wfslayers as those are the ones that can be used in analysis.